network/forks/fork.go

```diff
@@ -165,7 +166,19 @@ func SortedForkVersions(forkSchedule map[[4]byte]types.Epoch) [][4]byte {
 		i++
 	}
 	sort.Slice(sortedVersions, func(a, b int) bool {
-		return forkSchedule[sortedVersions[a]] < forkSchedule[sortedVersions[b]]
```

Comment:
when all the epochs are the same, we need to break ties with a lexicographic sort. TODO: make a mapping between the `Version` enum and the equivalent config attributes so that we can have an explicit ordering, and not rely on the convention of making later forks lexiographically "higher" than earlier ones. This is a footgun when setting up testnets.